Bottom: b3dc79cf8aff7dcd9f9b7e19de2a077d9c211304
Top:    b373a7c4b40168e9a7f85e06ceb65b2bc37ae12f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-30 19:52:20 -0400

Refresh of device-trees

---

diff --git a/drivers/of/base.c b/drivers/of/base.c
index 6765652..133908a 100644
--- a/drivers/of/base.c
+++ b/drivers/of/base.c
@@ -17,8 +17,6 @@
  *      as published by the Free Software Foundation; either version
  *      2 of the License, or (at your option) any later version.
  */
-#define DEBUG
-
 #include <linux/ctype.h>
 #include <linux/module.h>
 #include <linux/of.h>
diff --git a/drivers/of/gpio.c b/drivers/of/gpio.c
index 917d66f..7e62d15 100644
--- a/drivers/of/gpio.c
+++ b/drivers/of/gpio.c
@@ -11,8 +11,6 @@
  * (at your option) any later version.
  */
 
-#define DEBUG
-
 #include <linux/device.h>
 #include <linux/errno.h>
 #include <linux/module.h>
diff --git a/drivers/of/of_i2c.c b/drivers/of/of_i2c.c
index e1d99c0..f37fbeb 100644
--- a/drivers/of/of_i2c.c
+++ b/drivers/of/of_i2c.c
@@ -18,7 +18,6 @@
 #include <linux/of_irq.h>
 #include <linux/module.h>
 
-
 void of_i2c_register_devices(struct i2c_adapter *adap)
 {
 	void *result;
diff --git a/drivers/of/of_spi.c b/drivers/of/of_spi.c
index 4317938..6dbc074 100644
--- a/drivers/of/of_spi.c
+++ b/drivers/of/of_spi.c
@@ -6,8 +6,6 @@
  * tree.
  */
 
-#define DEBUG
-
 #include <linux/module.h>
 #include <linux/of.h>
 #include <linux/device.h>
